docs/diginstroom/sip/1.0/usecases/gigapixel-artwork.md

```diff
@@ -0,0 +1,540 @@
+| `cf9j41p15z_Kolom1_deelopname1_tiff.tiff`<br>`cf9j41p15z_Kolom1_deelopname2_tiff.tiff`<br>`...`<br>`cf9j41p15z_Kolom5_deelopname6_tiff.tiff` | A very high-resolution image split in multiple files in the TIFF media format containing a photoregistration of the 'Uur- en kalenderwijzeplaat'. |
+| `metadata.xml` | A metadata record describing the 'Uur- en kalenderwijzeplaat'. |
+
+The metadata record can contain the following information:
+
+- the title of the work;
+- a description of the work;
+- a local identifier (e.g., Inventarisnummer)
+- the date the work was created;
+- the dimensions of the work;
+- the name, birthdate and deathdate of the author;
+- a list of meemoo licenses;
+- some keywords
+- the md5 checksums of the media files;
+- rights credits
+- ...
+
+Some of the metadata above is supplied in English and Dutch.
+
+## Applying the core concepts
+
+Since the metadata only describes a single artwork, we can consider it as the single Intellectual Entity in the SIP.
+
+We can distinguish a couple of file sets (in the TIFF or PSB file format) that represent the work in some manner: an overview with frame, an overview without frame, a stiched high-resolution image, a high-resolution image in parts, and a camera calibration target recording.
+
+Since each set of files can have a meaning on its own (i.e. one could focus on one of the TIFF files to get an idea of what the painting looks like), they are split into separate representations. This results in the following application of the [core concepts]({{ site.baseurl }}{% link docs/diginstroom/sip/1.0/3_core-concepts.md %}):
+
+|_Intellectual Entity_|the painting 'The lamentation over the Dead Christ' |
+|_Representation 1_| the access copy of the painting with frame |
+|_Representation 2_| the access copy of the painting without frame |
+|_Representation 3_| the first archive master: a high-resolution stitched representation in TIFF format |
+|_Representation 4_| the second archive master: a high-resolution stitched representation in Adobe Photoshop Large Document Format |
+|_Representation 5_| the second archive master: a high-resolution non-stitched representation|
+|_Representation 6_| the calibration result|
+|_Files (rep. 1)_|the TIFF file: `cf9j41p15z_overzichtsopname_metlijst_tiff.tiff`|
+|_Files (rep. 2)_|the TIFF file: `cf9j41p15z_overzichtsopname_zonderlijst_tiff.tiff`|
+|_Files (rep. 3)_|the TIFF file: `cf9j41p15z_stitch_bigtiff.tiff` |
+|_Files (rep. 4)_|the PSB file: `cf9j41p15z_stitch_psb.psb`|
+|_Files (rep. 5)_|the TIFF files: `cf9j41p15z_Kolom1_deelopname1_tiff.tiff`, `cf9j41p15z_Kolom1_deelopname2_tiff.tiff`, ..., `cf9j41p15z_Kolom5_deelopname6_tiff.tiff`|
+|_Files (rep. 6)_|the TIFF file: `cf9j41p15z_target_tiff.tiff`|
+
+This case relies on the 2D profile because:
+
+- there is one IE and multiple representations;
+- the artwork can be described using a combination of DCTERMS and a Schema.org;
```

Comment:
```suggestion
- the artwork can be described using a combination of the [DCTERMS](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/) and [Schema](schema.org/) metadata models;
```

Reply:
Same comment as with the 2D usecase: perhaps this should be rephrased so that it is made clear that Schema is actually needed?